scripts/ae_integration_test.sh

```diff
@@ -30,15 +30,14 @@ if [ -z "${imageUnderTest}" ]; then
   exit 1
 fi
 
-# for local tests it makes sense sometimes to pin the deployment to an
-# active version as that will speed up the deployment, for CI/CD this feature
-# is not recommended
-readonly gaeDeploymentVersion=$2
-if [ "${gaeDeploymentVersion}" ]; then
-    DEPLOYMENT_OPTS="-v $gaeDeploymentVersion --no-promote"
-    DEPLOYMENT_VERSION_URL_PREFIX="$gaeDeploymentVersion-dot-"
-
+# If provided, pin to a specific version. Otherwise, generate a random new version ID to prevent
+# inadvertent collisions.
+gaeDeploymentVersion=$2
+if [ -z "${gaeDeploymentVersion}" ]; then
+  gaeDeploymentVersion=$(uuidgen)
 fi
+DEPLOYMENT_OPTS="-v $gaeDeploymentVersion --no-promote"
```

Comment:
Do we also need to make sure that the previous version is not stopped as well, or is that the default behavior anyway?

Reply:
I don't think it would make a difference if the version was stopped or not. What is your concern exactly?

Reply:
If the tests are run in parallel, they may stop each other's versions.

Reply:
Oh, right. I'll add the --no-stop-previous-version flag just to be sure.